vignettes/data_standardization.Rmd

```diff
@@ -268,8 +268,6 @@ kable(head(cordoleani_do_raw, 5))
 
 -   Defined `sample_id` as `cage_id`
 
--   ~~TO DO: should likely remove the two zooplankton density columns. It is weird that those values do not add up to total prey_density.~~
-
 -   mapped `Canal channel` and `Agriculture` to `agricultural canal`
 
 -   mapped `Channel` and `River Channel` to `perennial instream`
```

Comment:
please replace nested ifelse statements with case_when anywhere that you use them